packages/jellyfish-api-core/__tests__/category/icxorderbook/listOrders.test.ts

```diff
@@ -0,0 +1,328 @@
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import {
+  ICXGenericResult, ICXOfferInfo, ICXOrderInfo, ICXOrder, ICXOffer, ICXOrderStatus
+} from '../../../src/category/icxorderbook'
+import BigNumber from 'bignumber.js'
+import { accountBTC, accountDFI, checkBTCBuyOfferDetails, checkBTCSellOrderDetails, checkDFIBuyOfferDetails, checkDFISellOrderDetails, idDFI, setup } from './common.test'
+
+describe('Should test ICXOrderBook.listOrders', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup(container)
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  afterEach(async () => {
+    // cleanup code here
+  })
+
+  it('Should list all the orders', async () => {
```

Comment:
```suggestion
  it('should list all the orders', async () => {
```
can you standardize all these? (need to be consistence)